<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1257</width>
    <height>837</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1257</width>
    <height>837</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>40</y>
      <width>391</width>
      <height>759</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetFixedSize</enum>
     </property>
     <property name="horizontalSpacing">
      <number>3</number>
     </property>
     <property name="verticalSpacing">
      <number>17</number>
     </property>
     <item row="9" column="0">
      <widget class="QLabel" name="sixes">
       <property name="text">
        <string>Sixes</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="p1twos">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="options">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="18" column="2">
      <widget class="QPushButton" name="p2yahtzee">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="p1threes">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="Smallstraight">
       <property name="text">
        <string>Small Straight</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QPushButton" name="p2fives">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="9" column="2">
      <widget class="QPushButton" name="p2sixes">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QPushButton" name="p1bonus">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QPushButton" name="p1fullhouse">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QPushButton" name="p2threes">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="Fours">
       <property name="text">
        <string>Fours</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QPushButton" name="p1sixes">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QPushButton" name="p1smallstraight">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="Foak">
       <property name="text">
        <string>Four of a kind</string>
       </property>
      </widget>
     </item>
     <item row="19" column="0">
      <widget class="QLabel" name="totalscore">
       <property name="text">
        <string>TOTAL SCORE</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="Bonus">
       <property name="text">
        <string>Bonus</string>
       </property>
      </widget>
     </item>
     <item row="16" column="0">
      <widget class="QLabel" name="LargeStraight">
       <property name="text">
        <string>Large striaght</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QPushButton" name="p1foak">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="Twos">
       <property name="text">
        <string>Twos</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="Sum">
       <property name="text">
        <string>Sum</string>
       </property>
      </widget>
     </item>
     <item row="17" column="2">
      <widget class="QPushButton" name="p2chance">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="18" column="0">
      <widget class="QLabel" name="YAHZEE">
       <property name="text">
        <string>YAHTZEE</string>
       </property>
      </widget>
     </item>
     <item row="19" column="2">
      <widget class="QPushButton" name="p2totalscore">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="p2ones">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="15" column="2">
      <widget class="QPushButton" name="p2smallstraight">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="17" column="0">
      <widget class="QLabel" name="Chance">
       <property name="text">
        <string>Chance</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="p1fours">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="Fives">
       <property name="text">
        <string>Fives</string>
       </property>
      </widget>
     </item>
     <item row="17" column="1">
      <widget class="QPushButton" name="p1chance">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="10" column="2">
      <widget class="QPushButton" name="p2sum">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="FullHouse">
       <property name="text">
        <string>Full House</string>
       </property>
      </widget>
     </item>
     <item row="14" column="2">
      <widget class="QPushButton" name="p2fullhouse">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="Player2">
       <property name="text">
        <string>Player2</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QPushButton" name="p1fives">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="16" column="2">
      <widget class="QPushButton" name="p2largestraight">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="19" column="1">
      <widget class="QPushButton" name="p1totalscore">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="Ones">
       <property name="text">
        <string>Ones</string>
       </property>
      </widget>
     </item>
     <item row="18" column="1">
      <widget class="QPushButton" name="p1yahtzee">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="p2twos">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="Threes">
       <property name="text">
        <string>Threes</string>
       </property>
      </widget>
     </item>
     <item row="12" column="2">
      <widget class="QPushButton" name="p2toak">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QPushButton" name="p2fours">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="Player1">
       <property name="text">
        <string>Player 1</string>
       </property>
      </widget>
     </item>
     <item row="11" column="2">
      <widget class="QPushButton" name="p2bonus">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QPushButton" name="p1toak">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="QPushButton" name="p1largestraight">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QPushButton" name="p1sum">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="p1ones">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="13" column="2">
      <widget class="QPushButton" name="p2foak">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="Toak">
       <property name="text">
        <string>Three of a kind</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>340</y>
      <width>701</width>
      <height>117</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="_dice_1">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="_dice_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="_dice_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="_dice_4">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="_dice_5">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>748</width>
      <height>138</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="dice1">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resource.qrc">:/images/QuestionMarkBox.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="openExternalLinks">
        <bool>false</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::TextInteractionFlag::NoTextInteraction</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="dice2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resource.qrc">:/images/QuestionMarkBox.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="openExternalLinks">
        <bool>false</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::TextInteractionFlag::NoTextInteraction</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="dice3">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resource.qrc">:/images/QuestionMarkBox.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="dice4">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resource.qrc">:/images/QuestionMarkBox.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="dice5">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resource.qrc">:/images/QuestionMarkBox.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Rolldice">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>480</y>
      <width>411</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ROLL DICE</string>
    </property>
   </widget>
   <widget class="QLabel" name="indicateturn">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>321</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PLAYER 1 TURN</string>
    </property>
   </widget>
   <widget class="QLabel" name="indicatebox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>600</y>
      <width>681</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="text">
     <string>indicate box</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1257</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
